test/github_push_retest_test.go

```diff
@@ -0,0 +1,96 @@
+//go:build e2e
+// +build e2e
+
+package test
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/google/go-github/v55/github"
+	"github.com/openshift-pipelines/pipelines-as-code/pkg/apis/pipelinesascode/keys"
+	tgithub "github.com/openshift-pipelines/pipelines-as-code/test/pkg/github"
+	twait "github.com/openshift-pipelines/pipelines-as-code/test/pkg/wait"
+	"gotest.tools/v3/assert"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func TestGithubPushRequestGitOpsComments(t *testing.T) {
+	ctx := context.TODO()
```

Comment:
should not be in nightly?

Reply:
I did not add initially to see that e2e works on this PR changes

Reply:
ah sorry doing this to you but if you amend can you remove them from nightly, we'll keep there for a while and if they cause problems due of rate limit we will disable them.. i think tha'ts a better way to iterate, what do you think?

Reply:
Sure no issues

Updated PR by removing nightly env